webpack/assets/javascripts/react_app/components/Editor/components/__tests__/__snapshots__/EditorSettings.test.js.snap

```diff
@@ -18,7 +18,7 @@ exports[`EditorSettings renders EditorSettings 1`] = `
         >
           Syntax
         </div>
-        <Uncontrolled(Dropdown)
+        <ForwardRef
```

Comment:
please notice that this snap is already fixed in [your other PR](https://github.com/theforeman/foreman/pull/7162/files#diff-9ff6832b207b4729b975babe0669a85a)

Reply:
Not relevant in this PR